docs/read.md

```diff
@@ -197,16 +242,57 @@ function useRead<Document>(
 ): String;
 ```
 
+##### Derived Data Reads
+
+Derived data in Redux add vast complexity; not reduces it. Fortunately 
+derived data (aka computed data) in Read/Write is designed to be simple, easy 
+and performant.
+
+To start let's review the challenges in Redux proper. Redux's [`createSelector`](https://redux.js.org/usage/deriving-data-selectors#createselector-overview) 
+requires overly complex boilerplate. It's so easy to get wrong that the 
+documentation calls out how you will probably write it wrong. It's that the API
+designer's job? Should the design reduce complexity, provide flexibility/expressiveness so it's easy to so the right thing?
+
+Read/Write is built on data-driven, composable pieces. Learning one concept is 
+built upon with others. The hard work of simplification makes things easy and composable. 
+
+The heart of computed data in Read/Write just uses a partial read. Since the 
+component only pulls up the primitive data it needs a simple equality check
+inside useRead ensures the component will only trigger a React render check
+when data needed for the compute has changed. 
+
+To calculating derived data just pull the needed data and calculate like normal.
+```ts
+const effortType = useRead<Sprint, 'effortType'>(
+  {path: 'sprints', id: sprintId}, 
+  'effortType'
+);
+const taskEfforts = useRead<Task, 'effortLevel'>(
+  {path: 'tasks', where: ['sprintId', '==', sprintId]}, 
```

Comment:
If using a where clause shouldn't we be expecting more than one document returned?